Source/Core/VideoBackends/Software/SWVertexLoader.cpp

```diff
@@ -184,6 +182,39 @@ static void ReadVertexAttribute(T* dst, DataReader src, const AttributeFormat& f
   }
 }
 
+static void ParseColorAttributes(InputVertexData* dst, DataReader& src,
+                                 const PortableVertexDeclaration& vdec)
+{
+  auto SetDefaultColor = [](u8* color, int i) {
```

Comment:
```suggestion
  const auto set_default_color = [](u8* color, int i) {
```
maybe there should be an exception for lambdas that are used to structure code, but for now this should be using snake_case for consistency with the rest of the codebase